testcontainers-dapr/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.dapr</groupId>
+        <artifactId>dapr-sdk-parent</artifactId>
+        <version>1.12.0-SNAPSHOT</version>
+    </parent>
+
+	<artifactId>testcontainers-dapr</artifactId>
+	<name>testcontainers-dapr</name>
+	<description>Testcontainers Dapr Module</description>
+	<packaging>jar</packaging>
+
+	<properties>
+		<snakeyaml.version>2.0</snakeyaml.version>
+		<testcontainers.version>1.20.0</testcontainers.version>
+		<logback-classic.version>1.4.12</logback-classic.version>
+        <wiremock.version>3.0.1</wiremock.version>
```

Comment:
Any reason to use wiremock here? We are using org.mockito and I would prefer to keep consistency within the SDK.

Reply:
We used Wiremock because that is heavily used with Testcontainers

Reply:
If this needs to be part of the official Dapr SDK, then we need to have consistency in the project.

Reply:
As per our discussion, Wiremock is used for spinning up a mock HTTP server, which Mockito cannot do. So, we need it, I agree.